src/components/widgets/AddComment.vue

```diff
@@ -441,26 +441,35 @@ export default {
   },
 
   mounted() {
-    ;[
-      'drag',
-      'dragstart',
-      'dragend',
-      'dragover',
-      'dragenter',
-      'dragleave',
-      'drop'
-    ].forEach(evt => {
-      if (this.$refs.wrapper) {
-        this.$refs.wrapper.addEventListener(evt, e => {
-          e.preventDefault()
-          e.stopPropagation()
-        })
-      }
+    this.mode =
+      this.currentProduction.is_publish_default_for_artists &&
+      this.isCurrentUserArtist
+        ? 'publish'
+        : 'status'
+
+    this.$nextTick(() => {
+      ;[
+        'drag',
+        'dragstart',
+        'dragend',
+        'dragover',
+        'dragenter',
+        'dragleave',
+        'drop'
+      ].forEach(evt => {
+        if (this.$refs.wrapper) {
+          this.$refs.wrapper.addEventListener(evt, e => {
+            e.preventDefault()
+            e.stopPropagation()
+          })
+        }
+      })
     })
   },
 
   computed: {
     ...mapGetters([
+      'currentProduction',
```

Comment:
The `currentProduction` value may be incorrect on global pages (My Tasks, My Checks, News Feed, ...).

You can use `this.productionMap.get(eventData.project_id)` to retrieve the production of the current task.

Reply:
Thx. It's fixed.